alembic.ini

```diff
@@ -38,6 +38,8 @@ prepend_sys_path = ./src
 # are written from script.py.mako
 # output_encoding = utf-8
 
+path_separator=os
```

Comment:
Have you tested this both with a new DB and an existing DB migration to make sure it doesn't break anything?